www/components/MyST.tsx

```diff
@@ -0,0 +1,270 @@
+import * as md from "mdast";
```

Comment:
`import type` to get rid of warnings (same for `unist`), because these packages *only* expose type definitions and no implementations.